composer.json

```diff
@@ -11,11 +11,13 @@
     "license": "MIT",
     "require": {
         "php": "^7.2",
+        "ext-simplexml": "*",
```

Comment:
Can we lock this to a specific version?

Reply:
I don't believe so? This is dictated by the php version I am pretty sure - this was only added to stop PhpStorm squawking